client/src/pages/Login/LoginForm/useStyles.ts

```diff
@@ -4,14 +4,23 @@ const useStyles = makeStyles((theme) => ({
   form: {
     width: '100%', // Fix IE 11 issue.
     marginTop: theme.spacing(1),
+    display: 'flex',
+    flexDirection: 'column',
+    justifyContent: 'space-around',
+  },
+  textField: {
+    margin: 0,
+    borderWidth: 0.5,
```

Comment:
You can use the CSS Border - Shorthand Property here :)

Reply:
There were 3 of these issues. They are all fixed.